de.akaflieg_freiburg.cavok.add_hours_and_minutes.json

```diff
@@ -0,0 +1,28 @@
+{
+    "app-id": "de.akaflieg_freiburg.cavok.add_hours_and_minutes",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "addHoursAndMinutes",
```

Comment:
`addhoursandminutes`

Reply:
Thank you for spotting the issue! I fixed that.  -- And by the way, thanks for your awesome work (on a Saturday night!)